### PR TITLE
Added zh/2018-01-hant

### PR DESCRIPTION
**Pull request by imasaru**

### CODE REVIEW

File: create.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ var LOCALE_VARS = {
     "uk": "Урок",
     "ja": "日課",
     "zh": "每日课程",
+    "ht": "每日課程"
```
Comment (VitalikL) [inline create.js]: @imasaru I am pretty sure this will never be used, since `xxx-hant` will reside in `zh`

Comment (imasaru) [reply create.js]: @VitalikL I may not understand you correctly... The strings are different in Simplified and Traditional—at least kangxi-wise.

Comment (imasaru) [reply create.js]: @VitalikL I may not understand you correctly... The strings are different in Simplified and Traditional—at least kangxi-wise.

**Comment by imasaru** [comment]
@VitalikL I never tested it... Will we need to add synonyms? Does it pull them from Wikipedia or something like that?

**Comment by imasaru** [comment]
@VitalikL Is there anything else that you would want me to do with this?

**Comment by VitalikL** [comment]
@imasaru I am going to try releasing tonight!

**Comment by imasaru** [comment]
😄 👍

**Pull request merged**